blkn/src/Select/index.js

```diff
@@ -0,0 +1,145 @@
+// @flow
+import * as React from "react";
+import css from "styled-jsx/css";
+import ChevronDown from "react-icons/lib/fa/chevron-down";
+import classnames from "classnames";
+import Typography from "./../Typography";
+
+const ERROR_COLOR = "#d21c1c";
+const MIDGRAY_COLOR = "#7f91a8";
+const LIGHTGRAY_COLOR = "#bac7d5";
+const DARKGRAY_COLOR = "#46515e";
+const WHITE_COLOR = "white";
+
+const HASERROR_CLASS = "hasError";
+const UNSELECTED_CLASS = "UnSelected";
+
+type Option = {
+  value: string,
+  label?: string,
+  disabled?: boolean,
+  visible?: boolean, // eslint-disable-line react/no-unused-prop-types
+};
+type Props = {
+  disabled?: boolean,
+  error?: string,
+  label?: string,
+  onChange: (SyntheticInputEvent<HTMLSelectElement>) => any,
+  options: Array<Option>,
+  placeholder?: string,
+  required?: boolean,
+  value: string,
+};
+type State = {
+  value: string,
+};
+
+const style = css`
+  label {
+    font-family: Roboto;
+    line-height: 1.43;
+    position: relative;
+    color: ${DARKGRAY_COLOR};
+  }
+  select {
+    font-size: 16px;
+    border-radius: 3px;
+    background-color: ${WHITE_COLOR};
+    border: solid 1px ${LIGHTGRAY_COLOR};
+    line-height: 1.25;
+    padding: 12px 16px;
+    color: ${DARKGRAY_COLOR};
+    -moz-appearance: none;
```

Comment:
these prefix variants are not done automatically by `styled-jsx`?

Reply:
I couldn't find it in the documentation, but just confirmed it does by trying it. :+1: